SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -125,4 +134,6 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 	Worker_RequestId NextRequestId = 0;
 
 	ESpatialConnectionType ConnectionType = ESpatialConnectionType::Receptionist;
+    
+    LoginTokenCb    LoginTokenCb_ = NULL;
```

Comment:
Nit: We don't really use _ in the code base to differentiate between type and variable. Perhaps we could call this something like LoginTokenCbInstance or simply LoginTokenCallback.

Reply:
You should be able to remove the = NULL as well (we don't use NULL in the code base).

Reply:
We unfortunately do not have a full list of coding standards. Even if we did I doubt we would have actually added documentation on this. Unreal itself does not use _ to mark member variables I believe. My personal preference is to mark member variables with some prefix/suffix however we are trying to keep to Unreal code standards/style.

Reply:
Done